src/domains/DevTeam/devTeamMembers.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import teamMember1 from 'assets/teamMembers/teamMember1.jpg';
+import GithubIcon from '@material-ui/icons/GitHub';
+import { Description as ResumeIcon } from '@material-ui/icons';
+import LinkedInIcon from '@material-ui/icons/LinkedIn';
+import faker from 'faker';
+
+export default [
```

Comment:
Just add a todo here that this will eventually be an API call, otherwise every time we wanted to edit/add a team member we'd have to rebuild and redeploy the entire front end!  Maybe that could be another task that you do if you want to squeeze in more hours before the start of the quarter.  That's a relatively small task that I think you will know how to do. You could also add it to the mocks folder on the front end.

Reply:
Will work on this